server/hydrogen-render/render-hydrogen-to-string-fork-script.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+// Called by `child_process` `fork` in `render-hydrogen-to-string.js` so we can
```

Comment:
The different layers here are getting a bit complex and won't be easy to follow for a new person to the project.

 - `server/hydrogen-render/render-hydrogen-to-string.js`: Runs everything in a `child_process` via `fork`
     - `server/hydrogen-render/render-hydrogen-to-string-fork-script.js`: The script to run in `fork`
         - `server/hydrogen-render/render-hydrogen-to-string-unsafe.js`: Runs everything in a `vm` context to setup the browser-like globals
              - `shared/hydrogen-vm-render-script.js`: Renders Hydrogen to `document.body`

Reply:
I've renamed these with 1-4 to show the usage layers more obviously.

 - `1-render-hydrogen-to-string.js`
 - `2-render-hydrogen-to-string-fork-script.js`
 - `3-render-hydrogen-to-string-unsafe.js`
 - `4-hydrogen-vm-render-script.js`
 
 Better naming? This still isn't great but now at least an easy order is defined for people get some bearing.

Reply:
Updated in https://github.com/matrix-org/matrix-public-archive/pull/64 to separate the concerns of `child_process` runner and Hydrogen render